src/utilities.h

```diff
@@ -445,6 +445,25 @@ bool isStandardPrefixName(const std::string &name);
  */
 size_t getVariableIndexInComponent(const ComponentPtr &component, const VariablePtr &variable);
 
+/**
```

Comment:
Does this include indirect equivalences? And should the documentation explain that?

Reply:
The body function for that method is:
```
bool isSameOrEquivalentVariable(const VariablePtr &variable1,
                                const VariablePtr &variable2)
{
    return (variable1 == variable2) || variable1->hasEquivalentVariable(variable2, true);
}
```

So, yes, it does consider indirect equivalences. Still, I have updated the documentation in commit c253986.